src/Bridge/Doctrine/Orm/Filter/AbstractFilter.php

```diff
@@ -173,4 +175,28 @@ protected function extractProperties(Request $request) : array
 
         return $request->query->all();
     }
+
+    /**
+     * @param array property
```

Comment:
to me looks like an abnormality, hence an exception to raise it. If the developer wants to ignore it, he can always catch that error, but if you ignore it, he has no means to know about it.